manifests/samples/full_stack_fs.yaml

```diff
@@ -73,6 +73,8 @@ spec:
     enabled: true
   portal: {}
   registry: {}
+  trivy:
```

Comment:
@heww 

why not 

```
trivy: {}
```

empty storage is required?

Reply:
`storage` is required, not optional